src/borg/helpers/parseformat.py

```diff
@@ -377,6 +378,14 @@ class Location:
         |                                                   # or
         """ + optional_archive_re, re.VERBOSE)              # archive name (optional, may be empty)
 
+    win_file_re = re.compile(r"""
+        (?:file:\/\/)?                                      # optional file protocol
```

Comment:
do slashes need escaping by a backslash?

Reply:
Ah no, in Python they do not require escaping.